landbosse/model/CollectionCost.py

```diff
@@ -546,7 +555,13 @@ def create_ArraySystem(self):
 
         # Calculate total length of each cable type, and total cost that calculated length of cable:
         for idx, (name, cable) in enumerate(self.cables.items()):
-            total_cable_len = self.calc_total_cable_length(cable, self.input_dict['cable_specs'], self.output_dict['num_full_strings'], self.output_dict['num_partial_strings'], self.output_dict['trench_len_to_substation_km'], self.output_dict['perc_partial_string'][idx])
+            cable_specs = self.input_dict['cable_specs']
```

Comment:
Splitting the variables out of the input dictionary this way makes them easier to inspect in a debugger.